projects/npm-tools/packages/npm-scripts/src/scripts/format.js

```diff
@@ -18,9 +18,21 @@ const DEFAULT_OPTIONS = {
 };
 
 /**
- * File extensions that we want Prettier to process.
+ * File extensions that we allow Prettier to process.
  */
-const EXTENSIONS = ['.js', '.json', '.jsp', '.jspf', '.scss', '.ts', '.tsx'];
+const EXTENSIONS = [
+	'.js',
+	'.json',
+	'.jsp',
+	'.jspf',
+	'.md',
+	'.markdown',
```

Comment:
In Frontend Infra repos we just use `.md` but there are some `.markdown` in liferay-portal.